source/_integrations/buienradar.markdown

```diff
@@ -37,7 +37,7 @@ The `buienradar` camera platform uses [buienradar.nl](https://buienradar.nl/) as
 Internally, this integration uses the radar map image as [documented](https://www.buienradar.nl/overbuienradar/gratis-weerdata) on buienradar.nl.
 The downloaded image is cached to prevent Home Assistant from making a new request to buienradar.nl multiple times a minute when Home Assistant checks for new stills from the camera.
 
-The camera entity is added disabled by default and should first be enabled before it starts reading the camera images.
+The camera entity is disabled by default and should be enabled before it reads the camera images. The camera entity is enabled by going to Settings > Devices and Services > Entities. Search for "camera.buienradar". Click on the entity and, after that, the cogwheel. Then ensure that the slider under "Enabled" is on.
```

Comment:
This applies to many many entities (of Buienradar and many others), not just the camera entity.

This is a more generic instruction of Home Assistant, to me it feels odd to add it just here?

../Frenck

Reply:
That is an excellent point. I agree it's a more generic HA feature.
I tried looking for documentation through official channels, which I could then link to. Do you have a good idea of where to search for it in the official docs?

And you are right, all of them are disabled, except for weather.buienradar.

For ease of usage for newer users, in my opinion, it should be officially documented somewhere, which can then link to... Somewhat like https://github.com/home-assistant/home-assistant.io/blob/current/source/_includes/integrations/option_flow.md is used in some integrations.

Reply:
Thank you for the suggestion. There is now a small procedure in the _Common tasks_ section of the documentation, describing how to enable an entity.